docs/development.md

```diff
@@ -15,9 +15,124 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-### Backend development
+## Developement guide
 
-TODO
+## Requirements
+
+- JDK 8 (JDK 11 is recommended) 
+- Maven 3.5.x
+
+## Backend development
+
+### Prerequisites
+
+- Install [docker](https://docs.docker.com/)
+- For Mac OS X use [Docker for Mac](https://docs.docker.com/docker-for-mac/)
+- [Docker compose](https://docs.docker.com/compose/) is also required.
+
+### Build and start Log Search in docker container
+```bash
+# to see available commands: run start-logsearch without arguments
+cd docker
+./logsearch-docker build-and-run # build mvn project locally, build docker image, start containers
```

Comment:
Command is `logsearch-docker`, but comment above says `start-logsearch`.  Shouldn't they match?